myuw/templates/base.html

```diff
@@ -150,6 +151,26 @@ <h2 class="sr-only" id="app_messages_desktop_label">MyUW Announcements</h2>
         <!-- end message banner -->
         {% endblock %}
 
+	{% if display_onboard_message %}
+	{% block desktop_tour_banner %}
+        <!-- NTS: message banner based off the banner above (Desktop only) -->
+        <div class="myuw-banner-desktop app-messages" id="tour_messages_desktop" aria-labelledby="app_messages_desktop_label" role="complementary">
+            <div class="message-banner">
+                <h2 class="sr-only" id="app_messages_desktop_label">MyUW Announcements</h2>
+                <div id="message_banner_location">
```

Comment:
`<div id="message_banner_location">`
This ID is being used in both the desktop and the mobile popup code. IDs need to be unique. I'd recommend changing this to `<div id="message_banner_location_desktop">` and the other one `<div id="message_banner_location_mobile">`

Reply:
It's a leftover from the desktop message banner, so removed it. When looked up in Git, it didn't seem to be used.

Reply:
It looks like it is being used for the standard messages area that can be set from the admin tools. Might want to double check that those still work after your change. It looks like there were 4 divs with that same ID.